app/helpers/vacancies_helper.rb

```diff
@@ -107,12 +107,13 @@ def vacancy_about_school_label_organisation(vacancy)
   end
 
   def vacancy_about_school_hint_text(vacancy)
-    return I18n.t('helpers.hint.job_summary_form.about_schools') if vacancy.organisations.many?
+    if vacancy.organisations.many?
+      return I18n.t('helpers.hint.job_summary_form.about_schools',
+                    organisation_type: vacancy.parent_organisation.group_type.gsub!('_', ' '))
+    end
 
-    I18n.t(
-      'helpers.hint.job_summary_form.about_organisation',
-      organisation_type: organisation_type_basic(vacancy.parent_organisation).capitalize,
-    )
+    I18n.t('helpers.hint.job_summary_form.about_organisation',
+           organisation_type: organisation_type_basic(vacancy.parent_organisation).capitalize)
```

Comment:
I think we should either use organisation_type_basic or gsub in both cases (consistently).